aiida_orca/parsers/__init__.py

```diff
@@ -40,9 +37,12 @@ def parse(self, **kwargs):
         if fname_out not in out_folder._repository.list_object_names():  #pylint: disable=protected-access
             raise OutputParsingError('Orca output file not retrieved')
 
-        parsed_obj = io.ccread(os.path.join(out_folder._repository._get_base_folder().abspath, fname_out))  #pylint: disable=protected-access
-
-        parsed_dict = parsed_obj.getattributes()
+        try:
+            with self.retrieved.open(fname_out) as handler:
+                parsed_obj = ccread(handler.name)
+                parsed_dict = parsed_obj.getattributes()
+        except:  #pylint: disable=bare-except
```

Comment:
Do you have a concrete example where cclib throws? I am wary of this catch-all except, would rather have it more specific and we can improve it continually as we find more exception types when using the plugin. Then we can have better error messages.